extension/content_script.js

```diff
@@ -11,18 +22,7 @@ var timer = null;
 var speed = 10; // Base speed, not accounting for sentence length; adjustable w/ D/S
 var speed_bias = 500; // Minimum amount of speed spent on each sentence (in milliseconds)
 var speed_adj = 0; // Speed after it has been adjusted by sentence length
-var containerInit = 0; // Has container been initialized? 0 -> NO, 1 -> YES
 var firstMove = true; // Is this the first time it has been moved? True until after first movement. 
-var tracker_len = 0; // Length of current tracker (length of sentence or smaller chunk); 0 <= tracker_len
-var startReading = false; // Has the user started reading?
-// List of dom IDs that contain readable content.
-// Sorted in order of reading progression. 
-// E.g. $("#" + readableDomIds[0]) gets you the jQuery element to the first readable content.
-// Populated by parseDocument()
-var readableDomIds = [];
-// Pointer to an element in readableDomIds. The current container the tracker is in.
-// Will always be in [0, readableDomIds.length)
-var containerId = 0;
```

Comment:
Moved these guys up to be next to container because [container, containerId] have to be kept in sync.

Reply:
LGTM